apps/mobile/src/store/constants.ts

```diff
@@ -278,3 +278,47 @@ export const mockedChains = [
     transactionService: 'https://safe-transaction-arbitrum.safe.global',
   },
 ]
+
+export const STORAGE_IDS = {
```

Comment:
Should this be an enum?

Reply:
I don't see the change yet, so will unresolve the thread. Please generally don't resolve others' comments (let the author do it themselves) so that it's easier to follow up. :)

Once it's an enum, you can also use it as the type of the `key` in the functions below for extra type safety.